### PR TITLE
Configure renovate

### PR DESCRIPTION
**Pull request by ErikMichelson**

### CODE REVIEW

**Comment by DerMolly** [comment]
exactly what I was about to do

**Comment by DerMolly** [comment]
do we want to let renovate run during the night?

**Comment by ErikMichelson** [comment]
I thought so, I wanted to look at least at all the config options once. Therefore this is still WIP.

File: renovate.json [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+      ]
+    },
+    {
+      "extends": "packages:jsTest",
+      "groupName": "JS test packages"
+    }
+  ],
+  "schedule": [
+    "after 2am every weekday",
+    "before 8am every weekday"
```
Comment (DerMolly) [inline renovate.json:38]:
```suggestion
    "after 2am",
    "before 8am"
```

**Pull request merged**